### PR TITLE
Adds an admin secret to mass heal and revive everyone.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Turns out, the only way to do this, if desired as an admin, is through SQL. And that's lame!

So, this adds an admin secret to mass heal and revive everyone on the station with a mob. Not much else nuance to it really.

## Why It's Good For The Game

Turns out we've been asking about it in bus for like, 3-4 years now?
I was also asked about it in a round today, so here we go.

## Changelog

:cl:
admin: Admins have a new secret to mass revive and heal all players on the server.
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
This'll massively help my lazy ass when ~~dicking around killing spawned mobs on local~~ testing PRs in a responsible way.

File: code/modules/admin/verbs/secrets.dm [hunk 1]
```diff
@@ -604,6 +604,24 @@ ADMIN_VERB(secrets, R_NONE, "Secrets", "Abuse harder than you ever have before w
...
+		if("mass_heal")
+			if(!is_funmin)
+				return
+			var/heal_mobs = tgui_alert(usr, "Heal all mobs and return ghosts to their bodies?", "Mass Healing", list("Yes", "No", "Cancel"))
+			if(!heal_mobs || heal_mobs == "Cancel")
+				return
+
+			for(var/mob/dead/observer/ghost in GLOB.player_list) //Return all ghosts if possible
+				if(!ghost.mind || !ghost.mind.current) //won't do anything if there is no body
+					return FALSE
```
Comment (MrMelbert) [inline code/modules/admin/verbs/secrets.dm:616]: Why are we returning and not continuing here?

File: code/modules/admin/verbs/secrets.dm [hunk 2]
```diff
@@ -604,6 +604,24 @@ ADMIN_VERB(secrets, R_NONE, "Secrets", "Abuse harder than you ever have before w
 			message_admins("[key_name_admin(holder)] [ctf_controller.instagib_mode ? "enabled" : "disabled"] instagib mode in CTF game: [selected_game]")
 			log_admin("[key_name_admin(holder)] [ctf_controller.instagib_mode ? "enabled" : "disabled"] instagib mode in CTF game: [selected_game]")
 
+		if("mass_heal")
+			if(!is_funmin)
+				return
+			var/heal_mobs = tgui_alert(usr, "Heal all mobs and return ghosts to their bodies?", "Mass Healing", list("Yes", "No", "Cancel"))
+			if(!heal_mobs || heal_mobs == "Cancel")
```
Comment (MrMelbert) [inline code/modules/admin/verbs/secrets.dm:611]: You offer three options, "Yes" "No" "Cancel"

...But "No" doesn't do anything

**Comment by MrMelbert** [comment]
I think it would be funny if you played the "Hallelujah" sfx to everyone when you use it but that might just be me

File: code/modules/admin/verbs/secrets.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/modules/admin/verbs/secrets.dm:611]: chrome_8yDTSCDl0f

**Pull request merged**